src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/View/merchandise_returns.html.twig

```diff
@@ -37,10 +37,10 @@
       <tbody>
       {% for return in orderForViewing.returns.orderReturns %}
         <tr>
-          <td>{{ return.date|date_format_full }}</td>
-          <td>{{ return.type }}</td>
-          <td>{{ return.stateName }}</td>
-          <td class="text-right">
+          <td class="merchandise-return-date">{{ return.date|date_format_full }}</td>
```

Comment:
As I said in a previous PR, I'm more in favor of using jvascript const and identify it by a number instead of adding classes everywhere.

Reply:
If you use a number, it means that when we add a td, tests will be broken

I would use a class or id, but I would probably use a `.js-` prefixed class on anything used with JavaScript

Reply:
Done thx !

Reply:
I think `js-test-*` is the best pattern to choose